meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx_6.1.bb

```diff
@@ -1,3 +1,8 @@
+# We need to extend files paths because the -rt version of this
+# recipe includes this one and we need to have patches for this
+# recipe available in -rt recipe.
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```

Comment:
Won't PN be linux-lmp-fslc-imx-rt when using that recipe on both lines here? I had the impression that this one in particular required the full path instead of using PN as it gets included by imx-rt.

Reply:
@ricardosalveti I thought that way too, but in fact we have a correct FILESEXTRAPATHS :)
```
bitbake-getvar -r linux-lmp-fslc-imx-rt  FILESEXTRAPATHS
...
#   :prepend /home/cryo/work/foundries/lmp/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx_6.1.bb:1
#     "/home/cryo/work/foundries/lmp/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx:"
#   :prepend /home/cryo/work/foundries/lmp/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx-rt_6.1.bb:4
#     "/home/cryo/work/foundries/lmp/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx-rt:"
...
FILESEXTRAPATHS="/home/cryo/work/foundries/lmp/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx-rt:/home/cryo/work/foundries/lmp/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx:__default:"
```

Reply:
Nice, alright :-)